src/App.js

```diff
@@ -27,13 +27,15 @@ function App() {
             setCards={setCards}
             userInput={userInput}
             setUserInput={setUserInput}
+            handleDeleteCard={deleteCard}
           />
         </Route>
         <Route path="/addnew">
           <AddNewPage
             cards={cards}
             setCards={setCards}
             addNewCard={addNewCard}
```

Comment:
This function `onAddNewCard` comes from your `AddNew` component. Since it doesn't do anything in `AddNewPage` you should just keep the naming the same with `onSomethingSomething`. This way it would be much clearer.
```suggestion
            onAddNewCard={addNewCard}
```